ArduCopter/mode_sport.cpp

```diff
@@ -32,37 +32,33 @@ void ModeSport::run()
     update_simple_mode();
 
     // get pilot's desired roll and pitch rates
-
-    // calculate rate requests
-    float target_roll_rate_cds = channel_roll->norm_input_dz() * g2.command_model_acro_rp.get_rate() * 100.0;
-    float target_pitch_rate_cds = channel_pitch->norm_input_dz() * g2.command_model_acro_rp.get_rate() * 100.0;
+    float target_roll_rads, target_pitch_rads, target_yaw_rads;
+    get_pilot_desired_rates_rads(channel_roll->norm_input_dz(), channel_pitch->norm_input_dz(), channel_yaw->norm_input_dz(), target_roll_rads, target_pitch_rads, target_yaw_rads);
```

Comment:
I don't think this actually compiles with sport mode enabled because this function is a protected acro mode function.

Maybe have a search for this line and set it to 1
`# define MODE_SPORT_ENABLED 0`